cap/modules/experiments/utils/cadi.py

```diff
@@ -39,52 +37,21 @@
 from cap.modules.schemas.models import Schema
 
 from ..errors import ExternalAPIException
+from ..serializers import CADISchema
 from .common import generate_krb_cookie
 
-CADI_FIELD_TO_CAP_MAP = {
-    "name": "name",
-    "description": "description",
-    "contact": "contact",
-    "creatorDate": "created",
-    "URL": "twiki",
-    "PAPER": "paper",
-    "PAS": "pas",
-    "publicationStatus": "publication_status",
-    "status": "status",
-}
-
 
 def get_sso_cookie_for_cadi():
     """Get sso cookie needed to authenticate to CADI."""
     principal, kt = current_app.config['KRB_PRINCIPALS']['CADI']
-    url = current_app.config['CADI_AUTH_URL']
+    if principal is None or kt is None:
```

Comment:
would be better if we put this inside kinit decorator (https://github.com/cernanalysispreservation/analysispreservation.cern.ch/blob/ab70dca403d8a9fa6c92305947c3b63d3de4397d/cap/modules/experiments/utils/common.py#L40)
also, we could check just for false values (empty string will fail kinit command as well)